docusaurus/docs/users/howto/self-hosting.md

```diff
@@ -72,11 +86,13 @@ git@github.com:BobaBoard/selfhost-example.git
 
 ### flake.nix
 
-The `flake.nix` file is the backbone of your NixOS server configuration. It inclues directions to additional configuration instructions hosted publically on GitHub. (The nix language is unique, so don't be surprised if some of the syntax is especially unfamiliar.)
+The `flake.nix` file is the backbone of your NixOS server configuration. It includes directions to other publicly-available NixOS configurations on GitHub, which allow you to add new programs to your server without having to configure them yourself. Including directions to external resources also allows your server to easily stay up to date with whatever changes are made to those resources.
```

Comment:
"`flake.nix`" is not a good header. You're also not instructing to do anything in this section. 

I think you might want to rework this in maybe a "understanding the configuration structure" and then pure instructions.

This is kind of a running issue throughout the whole section: there's no way to easily separate what is instructions and what is explanations, and you can easily miss a step if you're skimming or not paying close attention. People very rarely read documentation end to end, so it needs to be mostly parsable at a glance.

Reply:
yeah definitely suffering here from just uncritically adopting devon's rough notes (not like devon was trying to do good information hierarchy or anything). ended up rearranging the whole thing so there's a clear "we're just going to look at the files/file structure and talk about why it's like that for a little bit here" section with no commands in it.